.github/workflows/finishReleaseCycle.yml

```diff
@@ -28,16 +36,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           ISSUE_NUMBER: ${{ github.event.issue.number }}
           COMMENT: |
-            This issue either has unchecked QA steps or has not yet been marked with the `:shipit:` emoji of approval.
+            Oops, something went wrong! This issue must:
+                1. Have unchecked QA steps, or
+                2. Not have been marked with the `:shipit:` emoji of approval, or
+                3. Have been closed by someone who is not a member of @Expensify/Mobile-Deployers.
             Reopening!
 
   # Update the production branch to trigger the production deploy.
   updateProduction:
     runs-on: ubuntu-latest
 
     # Note: Anyone with Triage access to the New Expensify repo can trigger a production deploy
-    needs: checkDeployBlockers
-    if: ${{ needs.checkDeployBlockers.outputs.hasDeployBlockers == 'false' }}
+    needs: validate
```

Comment:
Any reason this name is changing?

Reply:
Because it's not only checking deploy blockers anymore – it's validating the actor _and_ validating that there's no unresolved deploy blockers.